test/test-parse-expr.js

```diff
@@ -68,8 +68,19 @@ var assert = require('assert'),
     ],
     what: 'Triple backslash in quoted string (GH Issue #345)'
   },
-  { source: 'ENVELOPE ("Wed, 16 Dec 2015 16:19:25 +0100 (CET)" "Subject Line" ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL)) NIL NIL NIL "<message@id.with" <quote.angleBracket@and.space>"))',
+  { source: 'FLAGS (\\Seen)'
+        + ' ENVELOPE ("Wed, 16 Dec 2015 16:19:25 +0100 (CET)"'
+        + ' "Subject Line" ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL)) ((NIL NIL NIL NIL))'
+        + ' NIL NIL NIL "<message@id.with" <quote.angleBracket@and.space>")'
+        + ' INTERNALDATE "17-Jul-1996 02:44:25 -0700"'
+        + ' BODYSTRUCTURE (("TEXT" "PLAIN" ("CHARSET" "US-ASCII") NIL NIL "7BIT" 1152 23)'
+        + ' ("TEXT" "PLAIN" ("CHARSET" "US-ASCII" "NAME" "cc.diff")'
+        + ' "<960723163407.20117h@cac.washington.edu>" "Compiler diff"'
+        + ' "BASE64" 4554 73)'
+        + ' "MIXED")',
```

Comment:
updated this test to ensure data before & after `ENVELOPE` are parsed correctly.  No changes were needed to get this to pass, other than updating the results